src/Test/Utilities/Portable/project.json

```diff
@@ -1,7 +1,11 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
-    "System.Collections.Concurrent": "4.0.10"
+      "xunit": "2.1.0",
+      "Microsoft.NETCore": "5.0.0",
+      "Microsoft.NETCore.Portable.Compatibility": "1.0.0",
+      "System.Xml.XmlDocument": "4.0.0",
+      "System.Console": "4.0.0-beta-23123",
+      "System.Threading.Thread": "4.0.0-beta-23123"
   },
```

Comment:
Make note that this project was relying on packages to be inherited from `<ReferenceOutputAssembly>false</ReferenceOutputAssembly>` only projects. These are no longer inherited in this version of NuGet.